packages/utilities/psammead-test-helpers/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.2.1   | [PR#264](https://github.com/BBC/psammead/pull/264) Resolving package-lock issues. |
```

Comment:
```suggestion
| 0.2.2   | [PR#264](https://github.com/BBC/psammead/pull/264) Resolving package-lock issues. |
```

Reply:
As discussed separately this will become `0.3.1` following the merge of https://github.com/bbc/psammead/pull/258